tests/Tests/500__jupyterhub/test.robot

```diff
@@ -74,7 +74,30 @@ Can Launch Python3
    [Tags]  Sanity  TBC
    Launch Python3 JupyterHub
 
+Verify Failing Images Are Shown In Notifications
+    [Documentation]    Verifies failing Notebook names are shown in RHODS Dashboard Notifications
+    [Tags]    Sanity
+    ...       ODS-470
+    ${build_name} =    Search Last Build    namespace=redhat-ods-applications    build_name_includes=pytorch
+    Delete Build    redhat-ods-applications    ${build_name}
+    Verify Failed Build In Notifications
+    [Teardown]    Rebuild Pytorch
+
 
 *** Keywords ***
 JupyterHub Testing Suite Setup
   Set Library Search Order  SeleniumLibrary
+
+Verify Failed Build In Notifications
+    [Documentation]    Verifies Notifications show failed Notebook name
+    Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    Wait For RHODS Dashboard To Load
+    Click Element    xpath=//*[contains(@class,'notification-badge')]
+    Page Should Contain Element
+    ...    xpath=//*[contains(text(),'These notebook image builds have not started')]/../..//*[contains(text(),'PyTorch')]
```

Comment:
this message doesn't mean that there is a failed build. Please, refer to ODS-718 for testing the entire notification system

Reply:
Updated